include/lsst/afw/image/Calib.h

```diff
@@ -51,6 +52,29 @@ namespace cameraGeom {
     class Detector;
 }
 namespace image {
+
+static double const JanskysPerABFlux = 3631.0;
+
+/// Compute AB magnitude from flux in Janskys
+inline double abMagFromFlux(double flux) {
+    return -2.5*std::log10(flux/JanskysPerABFlux);
+}
+
+/// Compute AB magnitude error from flux and flux error in Janskys
+inline double abMagErrFromFluxErr(double fluxErr, double flux) {
+    return std::abs(fluxErr/(-0.4*flux*std::log(10)));
+}
```

Comment:
I'd prefer these to follow the same call signatures as the methods in `Calib`, that is, have a method that returns a `std::pair` of `flux` and `fluxErr` instead of one that returns just `fluxErr` (and the same for the inverse operations).